IlmBase/ImathTest/testFun.cpp

```diff
@@ -51,9 +51,24 @@ using namespace std;
     typedef long long unsigned int Int64;
 #endif
 
+template <typename To, typename From>
+static inline To
+bitwise_cast (From from)
+{
```

Comment:
I would recommend that we use the naming conventions of the coming C++20 std::bit_cast.
 https://en.cppreference.com/w/cpp/numeric/bit_cast so that eventually we can just transparently switch to std.

Reply:
Renamed to `bit_cast` and wrapped it in the `std` namespace, only `#if __cplusplus < 202002L`.

Reply:
Oh, no, sorry if I was confusing. Never put anything in std. I just meant "call it bit_cast".

Reply:
Though it's fine to do something like this:

    #if __cplusplus >= 202002L
        using std::bit_cast;
    #else
        template<...> bit_cast(...) {...}
    #endif

Reply:
I have now removed the cheeky `std` namespace.

Since `using namespace std` is already at the top of the file, I did not add the `using std::bit_cast` line.